src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -101,10 +101,8 @@ object RuleUtils {
         appendedBytesRatio < HyperspaceConf.hybridScanAppendedRatioThreshold(spark) &&
         deletedBytesRatio < HyperspaceConf.hybridScanDeletedRatioThreshold(spark)
 
-
       // For append-only Hybrid Scan, deleted files are not allowed.
-      lazy val isAppendOnlyCandidate = !hybridScanDeleteEnabled && deletedCnt == 0 &&
-        commonCnt > 0 &&
+      lazy val isAppendOnlyCandidate = deletedCnt == 0 && commonCnt > 0 &&
```

Comment:
The fix doesn't seem to align with the PR description: "For append-only hybrid scan, the threshold of deleted files doesn't matter."?

Could you also update the comment (line 104) to reflect the change (regarding the `hybridScanDeleteEnabled`)?

Reply:
I think the comment (line 104) is still valid - `deletedCnt == 0 ` 

Yes it doesn't matter but shouldn't be considered.. I updated the PR.

Reply:
I am still confused with the PR description: "we don't need to check the delete threshold." Where do we check the delete threshold for this change?

Reply:
It's because `hybridScanDeleteEnabled` is 
```scala
def hybridScanDeleteEnabled(spark: SparkSession): Boolean = {
    hybridScanDeletedRatioThreshold(spark) > 0.0
  }
```

Reply:
Ah sorry. My local repo had older version :)